library/src/scala/compiletime/ops/package.scala

```diff
@@ -196,22 +194,22 @@ package object ops {
      * val b: false ^ true = true
      * ```
      */
-    @infix type ^[X <: Boolean, Y <: Boolean] <: Boolean
+    type ^[X <: Boolean, Y <: Boolean] <: Boolean
```

Comment:
Why are these not infix?

Reply:
They don't need to be since they are symbolic anyway.